src/NHPersistenceCompatibilityTests/4.5/NHibernateSessionFactory.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Common;
+using NHibernate;
+using NHibernate.Tool.hbm2ddl;
+using NServiceBus.SagaPersisters.NHibernate.AutoPersistence;
+
+namespace Version_4_5
+{
+    public class NHibernateSessionFactory
+    {
+        public NHibernateSessionFactory()
+        {
+            SessionFactory = new Lazy<ISessionFactory>(Init);
```

Comment:
I think it was done to make the usage of `SessionFactory` inside `Persiter` more concise. /cc: @HEskandari